modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/StringUtils.java

```diff
@@ -16,22 +17,33 @@
 
 public class StringUtils {
 
+    /**
+     * Modify the cache size of the sanitizedNameCache, camelizedWordsCache and underscoreWords.
+     */
+    public static final String NAME_CACHE_SIZE_PROPERTY = "org.openapitools.codegen.utils.namecache.cachesize";
+    /**
+     * Modify the cache expiry of the sanitizedNameCache, camelizedWordsCache and underscoreWords.
+     */
+    public static final String NAME_CACHE_EXPIRY_PROPERTY = "org.openapitools.codegen.utils.namecache.expireafter";
```

Comment:
We might want to append `.seconds` or similar to this so the unit is self documenting.

Reply:
done